### PR TITLE
v2.7.0 blog post

### PR DESCRIPTION
**Pull request by siggisim**
---

**Version bump**: None

### CODE REVIEW

**Review by tylerwilliams — APPROVED** [review]
LGTM

File: website/blog/buildbuddy-v2-7-0-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  - A built-in Xcode locator in executors for discovering available Xcode versions
+
+
+## Coming soon!
+
+In addition to all of these great new features, we've also been laying the groundwork for some big new features that we're working on:
+
+ - A new caching infrastructure that will enable far more flexibility in where and how cache artifacts are stored.
+ - Mac support for [BuildBuddy Workflows](https://blog.buildbuddy.io/blog/meet-buildbuddy-workflows/).
+ - A managed cloud Bazel service that uses [Firecracker microVMs](https://firecracker-microvm.github.io/) that gives you warm, incremental Bazel instances in the cloud (right next to your RBE cluster) that spin up in milliseconds.
```
Comment (tylerwilliams) [inline website/blog/buildbuddy-v2-7-0-release-notes.md:60]:
```suggestion
 - A managed cloud Bazel service that uses [Firecracker microVMs](https://firecracker-microvm.github.io/) to give you warm, incremental Bazel instances in the cloud (right next to your RBE cluster) that spin up in milliseconds.
```

File: website/blog/buildbuddy-v2-7-0-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  - New BEP live data export functionality
+  - Flag overrides for platform properties for switching between pools without invalidating the cache
+  - SAML quality of life improvements
+  - Configurable executor docker image warm up 
+  - Support for Azure Storage as a storage backend
+  - AWS WebIdentity authentication support
+  - Private container registry support
+  - A built-in Xcode locator in executors for discovering available Xcode versions
+
+
```
Comment (tylerwilliams) [inline website/blog/buildbuddy-v2-7-0-release-notes.md:53]: nit: is the extra newline intentional?

Comment (siggisim) [reply website/blog/buildbuddy-v2-7-0-release-notes.md]: nope, removed

**Pull request merged**